### PR TITLE
Add join table for visits

### PR DESCRIPTION
**Pull request by Kakalanp**
- Adding new tables.
- Adding data to new tables
- Adding specific queries.

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -36,3 +36,37 @@ Set owner_id = 2 WHERE name IN ('Gabumon', 'Pikachu'),
 Set owner_id = 3 WHERE name IN ('Devimon', 'Plantmon'),
 Set owner_id = 4 WHERE name IN ('Charmander', 'Squirtle', 'Blossom'),
 Set owner_id = 5 WHERE name IN ('Angemon','Boarmon');
+
+INSERT INTO vets(name, age, date_of_graduation)
+VALUES ('William Tatcher', 45, 'Apr 23rd, 2000'),
+('Maisy Smith', 26, 'Jan 17th, 2019'),
+('Stephanie Mendez',64, 'May 4th, 1981'),
+('Jack Harkness', 38, 'Jun 8th, 2008');
```
Comment (Kingstalux) [inline data.sql]: - [x] Good job so far 👍 Kindly use the correct date format in adding data to your table. When I tried recreating your database, this date format kept giving errors. The correct format for `Apr 23rd, 2000` is `04-23-2000`. Kindly update all your dates using this format 👍 
- Your database:
Annotation 2022-04-28 052823

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @Kakalanp, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f744b797263304e546c4563475764546870342f67697068792e676966

## Highlights
✅ Good pull request description 👍 
✅ Good job adding the specialization and visit tables 👏 
✅ Good job answering all the queries 💯 


## Required Changes ♻️
_Check the comments under the review._
- [x] Also your format for updating the table kept giving errors. Kindly rectify this issue too 👍 
- Your database:
Annotation 2022-04-28 052910


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (Kakalanp) [reply data.sql]: Thank you very much! Fixing the issue right away!

**Review by Lordkaito — APPROVED** [review]
Hi @Kakalanp, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kakalanp** [comment]
Thanks!

**Pull request merged**